plugins/backstage-plugin-devcontainers-backend/package.json

```diff
@@ -23,15 +23,15 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.21.5",
+    "@backstage/backend-common": "^0.20.1",
     "@backstage/catalog-client": "^1.6.0",
     "@backstage/catalog-model": "^1.4.4",
     "@backstage/config": "^1.1.1",
     "@backstage/errors": "^1.2.3",
     "@backstage/plugin-catalog-common": "^1.0.21",
     "@backstage/plugin-catalog-node": "^1.7.2",
     "@types/express": "*",
-    "express": "^4.19.2",
+    "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
     "winston": "^3.2.1",
     "yn": "^4.0.0"
```

Comment:
GH won't let me go far down enough, but you might also be able to get rid of the backend-test-utils package. It accidentally got added in a previous commit, and I don't think we're using it

Reply:
Good to know, removed!